kore/src/Kore/Repl/Parser.hs

```diff
@@ -32,7 +32,10 @@ type Parser = Parsec String String
 commandParser :: Parser ReplCommand
 commandParser = do
     cmd <- commandParser0
-    (eof $> cmd) <|> (redirect cmd)
+    (eof $> cmd)
```

Comment:
I think we can make this clearer. For one, `commandParser0` could be something like `nonRecursiveCommand`.
Then we can extract `eof $> cmd` into `endOfInput`, then `try (pipe...)` into `pipeWithRedirect cmd`.